src/main/java/core/basesyntax/supplier/ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.supplier;
+
+import core.basesyntax.enums.Color;
+import java.util.Random;
+
+public class ColorSupplier {
+    public static String getRandomColor() {
```

Comment:
```suggestion
    public String getRandomColor() {
```

lets make this method not static